en/editor/components/spine.md

```diff
@@ -0,0 +1,234 @@
+
+4. Click the Preview button at the top of the editor to see the effect of vertex jitter of the Spine animation. For the code, please refer to the [SpineMesh](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/spine) example.
+
+## Spine Attachment Slots
+
+When using skeletal animation, you often need to attach a node on a part of the skeletal animation to achieve the effect of linking the node with the skeletal animation. Here is an example of how Spine can use attachment slots to attach stars to the dragon's tail, front claws and character's hands, and wiggle them along with the Spine node.
+
+![attach0](./spine/attach0.png)
+
+### Implement Spine Attachment Slots via Cocos Creator
+
+1. First create an empty node in the **Hierarchy** panel and rename it. Select the node and add the Skeleton component to the **Inspector** panel, drag and drop the asset into the SkeletonData property box of the Skeleton component and set the Skeleton component properties. 
+
+2. Then right-click on the Spine node in the **Hierarchy** panel and select **Create -> UI Render -> Sprite** to add a child Sprite node (tail, hand and claw nodes in the image below) to it.
+
+    ![attach1](./spine/attach1.png)
+
+3. Select the Spine node in the **Hierarchy** panel for which you want to set the attachment slot, and add the slot's information to the child node by setting Sockets in the **Inspector** panel (the value of Sockets represents the number of attachment slots).
+
+    ![attach2](./spine/attach2.png)
+
+4. Set the Path and Target of Sockets, select the location and child node object you want to attach respectively.
+
+    ![attach3](./spine/attach3.png)
+
+    You can see that a Sprite has been attached on the tail of the dragon in the **Scene** panel, and the same for the other attachment slots.
+
+    The same goes for the other attachment slots.
+    
+    ![attach4](./spine/attach4.png)
+
+5. Finally, drag the star asset to the Sprite component's `SpriteFrame` property. Save the scene and click the preview button at the top of the editor to see the star hanging on the dragon's tail and wiggling along with it. See the [SpineAttach](https://github.com/cocos-creator/test-cases-3d/tree/v3.0/assets/cases/spine) example in the example-case for details.
+
+## Spine Collision Detection
+
+The Spine attachment slots function allows you to detect the collision of a part of the skeleton animation. The following is an example of how Spine implements collision detection, by determining whether the character's feet are in contact with the ground or not to dynamically change the ground color when the character is running.
+
+![collider](./spine/collider0.png)
+
+1. First, you need to set the 2D physics engine to **Built-In Physics Engine** in the project settings.
+
+    ![collider](./spine/collider1.png)
+
+2. Create the Spine node and its attached child nodes as in the first two steps of the Spine attachment slots via the editor.
```

Comment:
as in the first two steps
-->
(as in the first two steps)

This sentence is a bit hard to understand though, perhaps We can simplify it.